index.html

```diff
@@ -22,24 +22,160 @@
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+      <div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <footer>
+       <ul class="work-4">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul> 
+        <div class="view-4">
+          <a href="#"><img src="images/Enabled.jpg"></a>
+        </div>
+      </footer>
+    </div>
+  </div>
+  <div class="page-5">
+    <div class="container-5">
+      <a href="#"><img src="images/Snapshoot Portfolio 4.jpg"></a>
+      <div id="text-5">
+        <a href="#"><img src="images/Project Title.jpg"></a>  
+      </div>
+        <ul class="column-5">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+      <div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <footer>
+       <ul class="work-5">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul> 
+        <div class="view-5">
+          <a href="#"><img src="images/Enabled.jpg"></a>
+        </div>
+      </footer>
+    </div>
+  </div>
 
+  <div class="page-6">
+    <section class="about-myself">
+      <h1>About Myself</h1>
+      <p>Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+      <a href="#">LET'S CONNECT</a>
+      <ul class="social-icons-2">
+        <li><a href="#"><img src="images/Twitter.jpg" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/Linkedin icon.jpg" alt="Linkedin"></a></li>
+        <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
+        <li><a href="#"><img src="images/Githhub.jpg" alt="Github"></a></li>
+      </ul>
+      <a class="link-1" href="#">Get my resume</a>
+    </section>
 
+    <div>
+      <ul class="others">
+        <li>Languages<a href="#"><span><img src="images/Union.jpg"></a></span>
+          <ul class="sub-lang">
+            <li><a href="3"><img src="images/Frame 9.jpg" alt="javaScript"></a></li>
+            <li><a href="3"><img src="images/Frame 10.jpg" alt="html"></a></li>
+            <li><a href="3"><img src="images/Frame 11.jpg" alt="css"></a></li>
+          </ul>
+        </li>
+        <li>Frameworks<a href="#"><span><img src="images/Union 2.jpg"></span></li>
+        <li>Skills<a href="#"><span><img src="images/Union 2.jpg"></span></li>
```

Comment:
Kindly adjust the design of the languages section to match the design template as closely as possible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199733512-c39c7e26-0996-48a4-8b8a-3c4a9b83a718.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/199733601-83c05159-9b93-47d8-bb4b-7db1ccb00bae.png)